projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/LLVMOptions.java

```diff
@@ -198,8 +199,12 @@ public static boolean isNativeCallStats() {
         return getParsedProperty(Property.NATIVE_CALL_STATS);
     }
 
+    public static boolean isNativeAnalysis() {
```

Comment:
Yes, I agree. I probably should also fix `isNativeAnalysisStats`, `isDiscoveryTestMode`, and `isPrintFunctionAsts` to have an `Enabled` postfix instead of the `is` prefix. Should I push the changes to this branch?